src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -1349,7 +1297,9 @@ impl<'a, 'tcx> Visitor<'tcx> for Checker<'a, 'tcx> {
                             }
                         }
                         _ => {
-                            // In normal functions no calls are feature-gated.
```

Comment:
Pull the content of the match arm out and run it after the abi match unconditionally. This way we also check that SIMD intrinsics with feature gates aren't usable in const fn

Reply:
Does platform intrinsics only contain simd intrinsics? Because this change hit to `wrapping_*` and others.

Reply:
> Does platform intrinsics only contain simd intrinsics?

Yes, only `simd_*` are platform intrinsics: https://github.com/rust-lang/rust/blob/488381ce9ef0ceabe83b73127c659e5d38137df0/src/librustc_typeck/check/intrinsic.rs#L388